client/components/takeQuiz/TakeQuiz.jsx

```diff
@@ -177,12 +185,103 @@ const TakeQuiz = (userId) => {
   }
 
   const submitAnswers = (userScore) => {
-    submitQuizAnswers(14, 1, userScore) // later change to: submitQuizAnswers(quizId, userId, userScore);
+    submitQuizAnswers({
+      correct_answer_count: userScore.correct,
+      incorrect_answer_count: userScore.incorrect,
+      id_quiz: quizId,
+      id_users: 11 // later change to: userId
+    })
     .catch(err => {
       console.error('Error: cannot submit quiz answers to database', err);
     })
   };
 
+  const getGlobalRankings = (userScore) => {
+    let rankings = [], allScores;
+    let percentage = (userScore.correct / userScore.total * 100).toFixed(0);
+    getQuizGlobalRankings(quizId)
+      .then(response => {
+        allScores = response.data.rows;
```

Comment:
Again, Kim and Tom told me to be a little more specific with what the response should actually be. In this case, it seems to be scores?